dso-l2/src/main/java/com/tc/l2/state/ConsistencyManagerImpl.java

```diff
@@ -263,7 +276,12 @@ public boolean isStuck() {
     @Override
     public Collection<Transition> requestedActions() {
       return getActions();
-    }        
+    }
+
+    @Override
+    public void forceTransition() {
+      ConsistencyManagerImpl.this.forceTransition();
```

Comment:
If you disable election rerun, you should force a new election here using [startElectionIfNecessary(ServerID.NULL_ID)](https://github.com/Terracotta-OSS/terracotta-core/blob/master/dso-l2/src/main/java/com/tc/l2/state/StateManagerImpl.java#L557)